### ISSUE TITLE
Add Numba RC Testing

### ISSUE THREAD

**Issue by seanlaw** [stumpy-dev/automate#1]
See https://numba.discourse.group/t/ann-numba-0-56-0-rc1/1417?u=seanlaw

Here is a simply script that hits the `pypi` (not pip) JSON API that will return ALL versions (including pre-release candidates):

```
#!/usr/bin/env python

import json
import sys
from urllib import request

def versions(pkg_name):
    url = f'https://pypi.python.org/pypi/{pkg_name}/json'
    releases = json.loads(request.urlopen(url).read())['releases']
    return releases

if __name__ == '__main__':
    print(*versions(sys.argv[1]), sep='\n')
```

Stick this into a `versions.py` file and then you can retrieve the latest release candidate (`rc`) via:

```
python versions.py numba | grep rc
```

or simply the latest release candidate with

```
python versions.py numba | grep rc | tail -n 1
```

**Comment by seanlaw** [comment]
See initial work in c0f311ff87a01908b6b029ef162b28a9e6318ed7

**Comment by seanlaw** [comment]
Fixed in c0f311ff87a01908b6b029ef162b28a9e6318ed7
